test/e2e/usersignup_test.go

```diff
@@ -31,43 +32,142 @@ func (s *userSignupIntegrationTest) TearDownTest() {
 	s.ctx.Cleanup()
 }
 
-func (s *userSignupIntegrationTest) TestUserSignupApproval() {
-	s.T().Run("no approval config - manual approval by default", func(t *testing.T) {
+func (s *userManagementTestSuite) TestAutomaticApproval() {
```

Comment:
Are we mixing two suites in this file? `userSignupIntegrationTest` and `userManagementTestSuite`

Reply:
I don't think so. This file contains tests for:
* automatic approval
* manual approval
* target cluster selection
* required verification
* username transformation

there is another file `user_management_test.go` that verifies everything regarding the user management - disabling, banning, deactivating.

Reply:
I'm confused. You are adding `func (s *userManagementTestSuite) TestAutomaticApproval() ` to `usersignup_test.go`
Should it be `func (s *userSignupIntegrationTest) TestAutomaticApproval()` instead?

Reply:
stupid me :man_facepalming:  - this slipped out of my notice. This is result of my bad copy-paste driven development. :man_facepalming: 
Thanks a lot for this catch :+1: